python/mxnet/optimizer/optimizer.py

```diff
@@ -105,6 +110,7 @@ def __init__(self, rescale_grad=1., param_idx2name=None, wd=0.,
         self._index_update_count = {}
         self.clip_gradient = clip_gradient
         self.multi_precision = multi_precision
+        self.aggregate_num = 0
```

Comment:
please add this in the parameter list in the class doc.

Reply:
It is not really a parameter though - it is up to the optimizer (not the user) to override this value if they support aggregated execution.